### ISSUE TITLE
Programatic configuration

### ISSUE THREAD

**Issue by adriannotimportant** [ical4j/ical4j#98]
My user has recently update their procedures for better virus control. When a treat is discovered they put the network in a partial lockdown that can last for days. During that connecting to Internet is restricted.
My application using iCal4J worked very well so far but during the lockdown periods the timezone update generates failures.
I suggested disabling the updates by changing the configuration file flag net.fortuna.ical4j.timezone.update.enabled to false but neither the client or I want to loose that capability forever. 
The only way to deal with this is to use a programmatic update of this property.
Would you consider adding that capability by allowing programmatic modification of properties on Configurator?
It would require a setProperty or OverrideProperty on the net.fortuna.ical4j.util.Configurator class.

Thank you,
Adrian

**Comment by benfortuna** [comment]
Yes, shouldn't be a problem to add this. Also note that I am currently looking to add caching support so you can control how often it will go to the Internet for updates. 

I will incorporate this change in that feature branch also.

Regards,
Ben

**Comment by adriannotimportant** [comment]
Thank you Ben
